ext/openssl/ossl_pkey_ec.c

```diff
@@ -1021,6 +1021,101 @@ static VALUE ossl_ec_group_set_seed(VALUE self, VALUE seed)
 
 /* get/set curve GFp, GF2m */
 
+
+
+/*
+ * call-seq:
+ *   group.field_type   => Symbol or Integer
+ *
+ * See the OpenSSL documentation for EC_METHOD_get_field_type().
+ * If the field type is GFp (NID_X9_62_prime_field) it will return :GFp
+ * If the field type is GF2m (NID_X9_62_characteristic_two_field) it will return :GF2m
+ * If OpenSSL returns any other value it will return the raw NID_X9_62 value
```

Comment:
As I commented in #425, I'd like to avoid the usage of `:GFp` or `:GF2m` symbols and simply use `OBJ_nid2sn()`.